### DISCUSSION TITLE
Does anyone know if we have `Region enum`? simiar like the Java SDK

### DISCUSSION THREAD

**Discussion by github2023spring** [awslabs/aws-sdk-rust#1185] [Q&A] [answered]
https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/regions/Regions.html

Is there an equivalent one?

**Comment by ysaito1001** [comment, marked as answer]
There is not an `Enum` represeting Regions, but we have [the Region struct](https://docs.rs/aws-types/latest/aws_types/region/struct.Region.html) instead in the `aws-types` crate.
